package.json

```diff
@@ -42,6 +42,7 @@
     "jade": "^1.11.0",
     "knex": "^0.13.0",
     "marked": "^0.3.6",
+    "moment": "^2.18.1",
```

Comment:
I think moment is already a dependancy. please rebase this branch off of origin/master